save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/fileuploader/CosvFileManagerComponent.kt

```diff
@@ -54,13 +58,13 @@ val cosvFileManagerComponent: FC<Props> = FC {
     @Suppress("GENERIC_VARIABLE_WRONG_DECLARATION")
     val organizationSelectForm = selectFormRequired<String>()
 
-    val (allAvailableFilesCount, setAllAvailableFilesCount) = useState(0L)
+    val (statistic, setStatistic) = useState<RawCosvFileStatisticsDto>()
```

Comment:
suggestion to create `RawCosvFileStatisticsDto.empty = RawCosvFileStatisticsDto(0, 0, 0, 0, ...)` and use it instead of checking on null

Reply:
Done